src/libraries/System.Threading.Channels/ref/System.Threading.Channels.csproj

```diff
@@ -14,5 +14,6 @@
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="$(MicrosoftBclAsyncInterfacesVersion)" />
```

Comment:
There's no need to use a prebuilt here. Please ProjectReference the live project instead. Same in the other projects.